opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/QueryStringQuery.java

```diff
@@ -59,7 +59,7 @@ public QueryStringQuery() {
         .put("phrase_slop", (b, v) -> b.phraseSlop(Integer.parseInt(v.stringValue())))
         .put("quote_field_suffix", (b, v) -> b.quoteFieldSuffix(v.stringValue()))
         .put("rewrite", (b, v) -> b.rewrite(v.stringValue()))
-        .put("type", (b, v) -> b.type(MultiMatchQueryBuilder.Type.parse(v.stringValue(),
+        .put("type", (b, v) -> b.type(MultiMatchQueryBuilder.Type.parse(valueOfToLower(v),
```

Comment:
Could you explain a little why this is the only one to be lowercased? And is parameter in OpenSearch DSL function case sensitive or not?

Reply:
Parameters on the OS side are enums and are case-sensitive. 
The problem is the OS side is inconsistent with their behaviour.  In some cases, the builders will lowerCase the values.  In other cases, they leave it up to the calling function. This is why only some parameters are lower cased here.

Reply:
For example: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/index/query/MultiMatchQueryBuilder.java#L599
The `type` parameter is lowerCased

Reply:
The type is lowercased in the `MultiMatchQuery` implementation through the `MultiMatchQueryBuilder.class`.
```
  public MultiMatchQueryBuilder type(Object type) {
    if (type == null) {
      throw new IllegalArgumentException("[multi_match] requires type to be non-null");
    } else {
      this.type = MultiMatchQueryBuilder.Type.parse(type.toString().toLowerCase(Locale.ROOT), LoggingDeprecationHandler.INSTANCE);
      return this;
    }
    ```
Adding toLower to type in QueryStringQuery makes it consistent case allowed across relevancy functions for type.